api.go

```diff
@@ -31,15 +34,18 @@ func NewClientWithConfig(config ClientConfig) *Client {
 // NewOrgClient creates new OpenAI API client for specified Organization ID.
 //
 // Deprecated: Please use NewClientWithConfig.
-func NewOrgClient(authToken, org string) *Client {
-	config := DefaultConfig(authToken)
+func NewOrgClient(authToken, org string) (*Client, error) {
+	config, err := DefaultConfig(authToken)
+	if err != nil {
+		return nil, err
+	}
 	config.OrgID = org
-	return NewClientWithConfig(config)
+	return NewClientWithConfig(config), nil
 }
 
 func (c *Client) sendRequest(req *http.Request, v interface{}) error {
 	req.Header.Set("Accept", "application/json; charset=utf-8")
-	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.config.authToken))
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.config.APIKey))
```

Comment:
It also needs to support api-key for authentication

Reply:
Oops, I forget here. I only changed the one in `newStreamRequest` .